particles/Tutorial/Javascript/Demo/Game.arcs

```diff
@@ -0,0 +1,161 @@
+// Tutorial Demo
+// Shows how to use Arcs features to create a tic-tac-toe game.
+import 'GameSchemas.arcs'
+import 'Stores.arcs'
+
+particle Game in './source/Game.js'
+  inout Person {name, avatar, id} playerOne
+  inout Person {name, avatar, id} playerTwo
+  in GameState {gameOver} gameState
+  inout Move {type, move} playerOneMove
+  inout Move {type, move} playerTwoMove
+  inout Move {type, move, playerAvatar} move
+  inout Move {type, move} event
```

Comment:
_event_ has a certain meaning, it's odd for it to be of type `Move`.  Suggest a different name here.

Reply:
Updated to have an actual Event type.